packages/patternlab/styleguide/source/assets/js/modules/accordions.js

```diff
@@ -1,28 +1,37 @@
 import checkActive from "../helpers/cssControlCode.js";
 
 export default (function (window,document,$,undefined) {
-  let $jsAccordion = $('.js-accordion');
-
+  let $jsAccordion = $('.ma__collapsible-content .js-accordion');
+  var $toggleLink = $('.ma__collapsible-content__toggle-all');
```

Comment:
```suggestion
  let $toggleLink = $('.ma__collapsible-content__toggle-all');
```

Reply:
Toggle link is a var because it's used inside other functions as well, for instance function `collapseLinkToggle`

Reply:
got it, I missed that